adapters/audienceNetwork/facebook.go

```diff
@@ -115,6 +128,50 @@ func (this *FacebookAdapter) makeAuthID(req *openrtb2.BidRequest) string {
 	return hex.EncodeToString(h.Sum(nil))
 }
 
+func dropSingleElement(extension []byte, elementName string, i interface{}) ([]byte, error) {
```

Comment:
Is it necessary to use `i` in this code? If you're removing an element by it's name, why does the type matter?

Reply:
OMG it works without `i`! Code is so much easier now! 
P.S. I swear I tried it already, but it didn't work :\